spec/features/admins/admin_can_delete_admin_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature "admin can delete admin", :js do
```

Comment:
do we need the `:js`?

Reply:
> do we need the `:js`?

i think its required so that line 16 `page.accept_alert` can run to interact with the confirmation dialog box